tests/test_check_obsfiles_ert_yaml.py

```diff
@@ -19,6 +19,8 @@
 _MIN_ERROR = 10
 _REL_ERROR = 0.05
 
+_RESAMPLING = None
```

Comment:
Instead of never testing the resampling, it would be better to create a test case that tests resampling based on the data provided by @tayloris when he created the original test.